### ISSUE TITLE
A11y: Add a descriptive and unique <title> element to each page

### ISSUE THREAD

**Issue by bacitracin** [CodeForPhilly/vacant-lots-proj#343]
## Describe the task
Screen reader users rely on the title to quickly learn the identity and purpose of the page. If you leave off the title, screen reader users will have to browse the page and infer the identity and purpose from the content, which is highly inconvenient.

Add a descriptive `<title>` to each page of the app, and make sure that it updates when you switch pages (this way it will announce for screenreader users)

Other guidelines:
- Unique info should come first in the title. (so it shows when people have multiple tabs open)
- Keep the title concise
- The title should be unique if possible.
- The page <title> should match or be very similar to the top heading in the main content

## Acceptance Criteria

- [ ] Every page has a valid <title> in the document <head>.

## Additional context

* https://dequeuniversity.com/rules/axe/4.4/document-title

**Comment by bacitracin** [comment]
@Amberroseweeks assigned, let me know if you have any questions!

**Comment by Amberroseweeks** [comment]
Thanks @bacitracin ! I am trying to get set up but this link is giving me a 404 error: https://github.com/CodeForPhilly/vacant-lots-proj/blob/main/SETUP

**Comment by Amberroseweeks** [comment]
got it thank you @nlebovits !
